src/utils/apollo-plugins.ts

```diff
@@ -0,0 +1,12 @@
+import { Context } from '../graphql/types/context';
+import { ApolloServerPlugin } from 'apollo-server-plugin-base';
+
+export const handleErrorsPlugin: ApolloServerPlugin<Context> = {
+	requestDidStart: () => {
+		return {
+			didEncounterErrors: (ctx) => {
+				console.error(ctx.errors);
```

Comment:
In a production application, do you think `console.log` is good or would you suggest something else?

Reply:
No, I would probably want to use a library to do this in a more flexible way. I was researching some of them before submitting and found that Winston is really popular and has some great functionalities (like supporting logging to multiple storages), but I didn't have the time to decide and implement it properly, so I left it out. I added a section on the "Next Steps" section of the docs because this is something that I would totally give more attention to for a production app.